fastlane/Fastfile

```diff
@@ -328,17 +332,17 @@ end
   # bundle exec fastlane build_and_upload_release skip_confirm:true
   # bundle exec fastlane build_and_upload_release beta_release:true
   #####################################################################################
-  desc "Builds and uploads for distribution"
-  lane :build_and_upload_release do | options |
+  desc 'Builds and uploads for distribution'
+  lane :build_and_upload_release do |options|
     ios_build_prechecks(skip_confirm: options[:skip_confirm],
-      internal: false,
-      internal_on_single_version: options[:beta_release],
-      external: true)
+                        internal: false,
+                        internal_on_single_version: options[:beta_release],
+                        external: true)
```

Comment:
💄  Same as above: I'd suggest to go to the next line after the opening parenthesis of `ios_build_prechecks(` to reduce the indentation level of the parameter lines.

Reply:
Fixed in https://github.com/Automattic/simplenote-ios/commit/a15e5d22d216e41b22583777cdb3e5ce79235de1